framework/source/class/qx/test/core/Validation.js

```diff
@@ -221,9 +259,12 @@ qx.Class.define("qx.test.core.Validation",
       var model = this.__model;
 
       // test some wrong inputs (allowed are male and female)
-      this.assertException(function() {model.setArray(0.999999999);}, qx.core.ValidationError, null, "A 0.999999999 is not in ['male', 'female']");
-      this.assertException(function() {model.setArray("malle");}, qx.core.ValidationError, null, "'malle' is not in ['male', 'female']");
-      this.assertException(function() {model.setArray("");}, qx.core.ValidationError, null, "A empty string is not in ['male', 'female']");
+      this.assertException(
+        function() {model.setArray(0.999999999);}, qx.core.ValidationError, null, "A 0.999999999 is not in ['male', 'female']");
```

Comment:
From experience I can only recommend to put formatting changes in "whitespace-only" commits. When you mix them with real changes it is very hard to review those commits.